setup.py

```diff
@@ -1,17 +1,13 @@
 #! /usr/bin/env python
 import codecs
-import sys
 from setuptools import setup
 
 __version__ = '1.0'
 
 
-if sys.version_info >= (3, ):
-    with codecs.open('README.rst', encoding='utf-8') as f:
-        long_description = f.read()
-else:
-    with open('README.rst') as f:
-        long_description = f.read()
+with codecs.open('README.rst', encoding='utf-8') as f:
+    long_description = f.read()
```

Comment:
Hey, what was the reason for this change? I recall there were issues with pip/setuptools/distutils and non-ascii text in long_description, and using unicode in Python 3 and bytes in Python 2 was the closest thing to make it work reliably.

Reply:
I see, thanks. I mistakenly thought this was a workaround for Python 2.6, but it sounds like it is necessary for Python 2.7. I have reverted these lines with the latest revision.